examples/values-facebook.yaml

```diff
@@ -25,6 +26,9 @@ routing:
   inferencePool:
     create: true
 
+  inferenceModel:
+    create: true
+
```

Comment:
Is the need just a side effect of a particular version of kgateway we are using?

Reply:
I don't think it hurts to include it, because that means it works for Kgateway and Istio. If we omit, it will only work for certain gateways.

Reply:
can we add a comment here in `values-facebook.yaml` and we can leave it at that?